babycenter/static/css/modules/_article-list.scss

```diff
@@ -13,7 +14,8 @@
 
 .article-list__title {
   font-weight: bold;
-  display: inline;
+  display: block;
+
 
   &:after {
```

Comment:
It doesn't make sense because it's a separator if it's one line. I will remove it.